website/content/en/preview/AWS/provisioning.md

```diff
@@ -341,3 +341,40 @@ Refer to general [Kubernetes GPU](https://kubernetes.io/docs/tasks/manage-gpus/s
 * `amd.com/gpu`: [AMD GPU device plugin for Kubernetes](https://github.com/RadeonOpenCompute/k8s-device-plugin)
 * `aws.amazon.com/neuron`: [Kubernetes environment setup for Neuron](https://github.com/aws/aws-neuron-sdk/blob/master/neuron-deploy/tutorials/tutorial-k8s.rst)
 {{% /alert %}}
+
+### Pod ENI (Security Groups for Pods)
+[Pod ENI](https://github.com/aws/amazon-vpc-cni-k8s#enable_pod_eni-v170) is a feature of the AWS VPC CNI Plugin which allows an Elastic Network Interface (ENI) to be allocated directly to a Pod. When enabled, the `vpc.amazonaws.com/pod-eni` extended resource is added to supported nodes. The Pod ENI feature can be used independently, but is most often used in conjunction with Security Groups for Pods.  Follow the below instructions to enable support for Pod ENI and/or Security Groups for Pods in Karpenter.
+
+{{% alert title="Note" color="primary" %}}
+You must enable Pod ENI support in the AWS VPC CNI Plugin before enabling Pod ENI support in Karpenter.  Please refer to the [Security Groups for Pods documentation](https://docs.aws.amazon.com/eks/latest/userguide/security-groups-for-pods.html) for instructions.
+{{% /alert %}}
+
+Once Pod ENI support is enabled in the AWS VPC CNI Plugin, you can enable Pod ENI support in Karpenter.  This can be handled via an environmental variable, command line flag, or via Helm.
```

Comment:
Thoughts on exposing this through the karpenter-settings configmap instead? I'm hesitant to use the word "OR" unless we have to. One way to do this is to keep the docs simple, even if we happen to have a little more powerful support under the hood.

Reply:
So you mean to keep the helm `enablePodENI` value, but populate the config map instead of pass an ENV, right?  Or am I misinterpreting.

I think we could remove the other config options in Preview docs, and leave just Helm.  For previous versions of docs we would need to provide at least one of the config options (flag or env), though.

Reply:
This is now in the global settings configmap